### ISSUE TITLE
showParentLevel not working when preceded by breadcrumb widget

### ISSUE THREAD

**Issue by timmetj** [algolia/instantsearch.js#3325]
Hello,

When a hierarchicalMenu widget and a breadcrumb widget are used, using the same attributes, then the showParentLevel false doesn't work, stays default true.

If you remove the breadcrumb widget, the showParentLevel false works again.

Expected to also hide the parents when the option is set to false

**Comment by Haroenv** [comment]
Did you add the parameter to both?

**Comment by timmetj** [comment]
Yes, Even though the option is not declared for the breadcrumb, but no difference

**Comment by timmetj** [comment]
Aha, found the issue then, the position of the widget seems to be it. When you first add the breadcrumb, and then the menu, it doesn't work:
https://codesandbox.io/s/w6x88pv3w5

**Comment by Haroenv** [comment]
Aha, that's right, sounds like a bug. Thanks for reporting it!
